docs/registry/create/_index.md

```diff
@@ -1276,13 +1280,95 @@ For more information on building a module in C++, see the [C++ SDK Build Documen
 {{% /tab %}}
 {{% /tabs %}}
 
+## (Optional) create a README
+
+To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:
+
+{{% expand "Click to view template" %}}
+
+Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.
```

Comment:
I think it makes sense to indicate variables that need replacing in the same way we do across code samples in the docs.